lib/foreman_rh_cloud/engine.rb

```diff
@@ -37,8 +37,13 @@ class Engine < ::Rails::Engine
         role 'ForemanRhCloud', [:view_foreman_rh_cloud]
 
         # Adding a sub menu after hosts menu
-        sub_menu :top_menu, :foreman_rh_cloud, :caption => N_('RH Cloud'), :icon => 'fa fa-cloud-upload' do
-          menu :top_menu, :level1, :caption => N_('Inventory Upload'), :url_hash => { controller: :'foreman_inventory_upload/react', :action => :index}
+        sub_menu :top_menu, :rh_cloud, :caption => N_('RH Cloud Services'), :icon => 'fa fa-cloud' do
```

Comment:
nit: name is a bit long

![Selection_311](https://user-images.githubusercontent.com/26363699/80683314-b2720600-8acc-11ea-91b2-cec6db055301.png)

how about Cloud Services?
![Selection_312](https://user-images.githubusercontent.com/26363699/80683700-4a6fef80-8acd-11ea-8d3e-f17c944d7b9a.png)

Reply:
Sounds good, but a bit unclear what cloud services do we mean. I suggest "Cloud services", but with RH logo instead of the cloud. Can we arrange it?

Reply:
@rohoover @MariSvirik do you have any ideas?
I know we are using font-awesome old free version where the RH icon doesn't exist,
is there a RH icon we can use?